packages/babel-parser/test/fixtures/es2017/async-functions/await-binding-inside-arrow-params-inside-async-arrow-params/input.js

```diff
@@ -0,0 +1 @@
+async (a = ({ await }) => {}) => {};
```

Comment:
I actually would have read this as a `ReferenceError` at first but after paying closer attention the early error makes sense 🤔